packages/teleport/src/components/BannerList/BannerList.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { useEffect, useState } from 'react';
+
+import { Box } from 'design';
+
+import { Banner } from './Banner';
+
+import type { Severity } from './Banner';
+
+export const BannerList = ({ banners = [] }: Props) => {
+  const [bannerList, setBannerList] = useState<{ [id: string]: BannerType }>(
+    {}
+  );
+
+  useEffect(() => {
+    const newList = {};
+    banners.forEach(banner => (newList[banner.id] = banner));
+    Object.assign(newList, bannerList);
+    setBannerList(newList);
+  }, [banners]);
```

Comment:
What will happen if we fetch alerts for one cluster and then the user will switch to another cluster? Will it display the merged list of alerts from two clusters? Is it intended?

Reply:
That is what would happen, yeah. 

We don't have a spec as to what's supposed to happen in all of these various notification edge cases. I'd like to see this notification system be replaced by a more fully featured notification system which could filter and categorize the notifications by location/type. For now though I just wanted it to merge any new notifications with ones that were already there.